### PR TITLE
DEMO-108: Add example usage of design engine

### PR DESCRIPTION
**Pull request by bdunogier**
> **Not for merge**

Demo integration of ezplatform-design-engine.

### CODE REVIEW

**Comment by andrerom** [comment]
@SylvainGuittard Seem to work, did some cleanup and slimmed down + improved theme.css example.

Open site with this and you can swap between `/site` and `/alternative_site` which has green color usage instead of orange on links and call for action stuff.

File: app/Resources/views/themes/demo/layout/header.html.twig [hunk 1]
```diff
@@ -7,12 +7,12 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
-            <a class="navbar-brand" href="/"><img src="{{ asset('assets/images/meadow.png') }}" class="img-responsive meadow-logo-top" /></a>
+            <a class="navbar-brand" href="{{ path( ezpublish.rootLocation() ) }}"><img src="{{ asset('assets/images/meadow.png') }}" class="img-responsive meadow-logo-top" /></a>
```
Comment (SylvainGuittard) [inline app/Resources/views/themes/demo/layout/header.html.twig]: According to the [doc](https://github.com/ezsystems/ezplatform-design-engine/blob/master/doc/assets.md) it should be something like that:
```
{{ asset("images/meadow.png", "ezdesign") }}
```
Am I wrong?

File: app/Resources/views/themes/demo/pagelayout.html.twig [hunk 2]
```diff
@@ -14,21 +14,21 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <link rel="Shortcut icon" href="{{ asset('assets/images/favicon.ico') }}" type="image/x-icon" />
```
Comment (SylvainGuittard) [inline app/Resources/views/themes/demo/pagelayout.html.twig]: Maybe:
```
{{ asset("images/favicon.ico", "ezdesign") }}
```

File: app/Resources/views/themes/demo/layout/header.html.twig (hunk 1 above)
Comment (andrerom) [reply app/Resources/views/themes/demo/layout/header.html.twig]: Indeed, the logo should be themable

**Comment by andrerom** [comment]
@SylvainGuittard updated

**Comment by SylvainGuittard** [comment]
Thanks André

**Comment by andrerom** [comment]
@kmadejski what was intended to be further done here in order to get this merged? _(ref @SylvainGuittard 's comment during demo regarding v1 demo work on using design engine)_

**Comment by kmadejski** [comment]
@andrerom for now just rebase and re-check if every template/asset is covered and used with `@ezdesign`. It will be finished soon.

**Comment by kmadejski** [comment]
@andrerom @SylvainGuittard rebased. No more changes needed, so if you approve then PR will be ready to merge.

**Comment by SylvainGuittard** [comment]
Thanks @kmadejski for the rebase. Already approved!
@damianz5 final review for you :)

**Pull request merged**